src/js/components/leftPanel/layersPanel/DateRange.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, ChangeEvent } from 'react';
+
+import { LayerProps } from 'js/store/mapview/types';
+
+interface DateRangeProps {
+  layer: LayerProps;
+}
+
+const DateRange = (props: DateRangeProps): JSX.Element => {
+  const { layer } = props;
+  console.log('<DateRange/>', layer);
+
+  const returnDateToday = (): string => {
+    const dateToday = new Date().toLocaleDateString();
+    const [monthToday, dayToday, yearToday] = dateToday.split('/');
+    const dayTodayFormatted = dayToday.length === 1 ? `0${dayToday}` : dayToday;
+    const monthTodayFormatted =
+      monthToday.length === 1 ? `0${monthToday}` : monthToday;
+    const dateTodayFormatted = `${yearToday}-${monthTodayFormatted}-${dayTodayFormatted}`;
+
+    return dateTodayFormatted;
+  };
+
+  const [startDate, setStartDate] = useState(returnDateToday());
+  const [endDate, setEndDate] = useState(returnDateToday());
```

Comment:
start and end dates are always the same? is this something on the todo list for the next pr? because PROD app does have diffrent start/end dates.

Reply:
@vaidotasp this is something I'll be attempting to implement in the next PR. I couldn't figure out how PROD dates were determined, so I'll continue digging into this and will keep you posted!

Reply:
👍